lib/tasks/sharetribe.rake

```diff
@@ -7,6 +7,15 @@ namespace :sharetribe do
     end
   end
 
+  namespace :landing_pages do
+    desc "Install sample landing page into initializers/landing_page.rb"
+    task :install_static => :environment do
+      source = File.join(Rails.root, "app", "services", "custom_landing_page", "landing_page.rb.template")
```

Comment:
a newcomer to ShareTribe OS might be confused why static landing pages don't work after running this rake task. Some output to remind setting `clp_static_enabled: true` might be helpful

Reply:
Added note in ec4170606b687f1e0bccc1bb33a3b9e89a7f7274